### ISSUE TITLE
[A11y]: Content in AI toggle tip not visible or scrollable at smaller resolution

### ISSUE THREAD

**Issue by andyl309** [carbon-design-system/carbon-ai-chat#628]
### Package version

v1.1.0

### Automated testing tool and ruleset

N/A

### Description

AI toggle tip is not scrollable at smaller resolution. See https://ibm-systems-z.slack.com/archives/C07RV4MPJNL/p1761930439815059

### Reproduction / example

N/A

### Steps to reproduce

-Insert huge amount of texts into AI toggle tip then shrink screen resolution
-Observe that when AI toggle tip is opened, there is no scrollbar so some contents are not accessible to user

### Suggested severity

None

### Code of conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-ai-chat/issues) for duplicate problems
